packages/axios-extension/src/auth/uaa.ts

```diff
@@ -257,4 +299,14 @@ export class Uaa {
         this.log.info('Got access token successfully');
         return response?.data?.access_token;
     }
+
+    /**
+     * @returns an access token using the BTP UAA credentials
+     */
+    public async getAccessTokenUsingClientCredentials(): Promise<string> {
```

Comment:
```suggestion
    public async getAccessTokenWithClientCredentials(): Promise<string> {
```
`With` sounds a bit better?